autoware_launch/launch/autoware.launch.xml

```diff
@@ -36,7 +36,7 @@
   <arg name="launch_web_controller" default="true" description="launch web controller"/>
 
   <!-- Global parameters -->
-  <!-- Do not add "group" in order to propagate global parameters -->
+  <!--  <group scoped="false"> -->
   <include file="$(find-pkg-share global_parameter_loader)/launch/global_params.launch.py">
```

Comment:
Thank you. I meant to add the group tag here.

```suggestion
  <group scoped="false">
    <include file="$(find-pkg-share global_parameter_loader)/launch/global_params.launch.py">
```

Reply:
I applied

Reply:
Sorry I missed opossite tag

Reply:
Thanks! But could you also fix the end of the tag?